files/default/elasticsearch-logrotate.conf

```diff
@@ -0,0 +1,8 @@
+/var/log/elasticsearch/*.log {
+    daily
+    missingok
+    rotate 10
+    compress
+    notifempty
+    copytruncate
+}
```

Comment:
Nitpick - please add a newline to this file. Some things can be super picky about this.